lib/experimental/class-wp-nav-menu-conversion-gutenberg.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * WP_Nav_Menu_Conversion_Gutenberg class
+ *
+ * @package gutenberg
+ * @since 6.3.0
+ */
+
+/**
+ * Manages Fallback behavior for Navigation menus.
+ *
+ * @access public
+ */
+class WP_Nav_Menu_Conversion_Gutenberg {
```

Comment:
Maybe a more illustrative name is `WP_Classic_To_Block_Menu_Converter`

Reply:
Yes that would be better 👍

Reply:
✅